esmvaltool/diag_scripts/autoassess/land_surface_soilmoisture/soilmoisture.py

```diff
@@ -117,23 +143,66 @@ def land_sm_top(run):
         dff = vol_sm1_run - ecv_clim
 
         # save output and populate metric
-        iris.save(dff, os.path.join(run['dump_output'],
-                                    'soilmoist_diff_{}.nc'.format(season)))
-        name = 'soilmoisture MedAbsErr {}'.format(season)
+        iris.save(dff, os.path.join(work_dir,
```

Comment:
Thanks for adding the provenance for the other files @alistairsellar! I noticed there is still no provenance for the files saved here though. Could you add that as well?

Reply:
Sorry, I forgot about the intermediate files. Thanks for spotting. Provenance now added for these.